src/context/LayoutContext.js

```diff
@@ -5,7 +5,7 @@ import {
 const LayoutContext = createContext({
   layout: {},
   setLayout: () => {},
-  getLayoutForKey: () => {},
+  getLayoutForKey: (key, defaultValue) => defaultValue,
```

Comment:
Do we need to be that explicit at this stage?

Reply:
You are right, we can mock it in the tests instead. I went ahead and did that